### DISCUSSION TITLE
What happened to the ability to use it for Personal Use?

### DISCUSSION THREAD

**Discussion by Zero3K** [Genymobile/feedback#24] [Genymotion Desktop] [answered]
I didn't see that option available on the first run of it. Please add it back soon.

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @Zero3K,

We are sorry but Personal Use is still available with Genymotion Desktop 3.3.1.
Please contact our technical support for help at https://support.genymotion.com/hc/en-us/requests/new

Thank you.
Best regards,
